### PR TITLE
HomeWork sql(26-32)

### PR DESCRIPTION
**Pull request by DadizhaO**

### CODE REVIEW

File: src/main/resources/sql/QuarryHM.sql [hunk 1]
```diff
@@ -0,0 +1,283 @@
+SELECT
+    order_num,
+    amount,
+    company,
+    credit_limit
+FROM
+    orders      ord
+    JOIN customers   cus ON ord.cust = cus.cust_num;
```
Comment (SerhiiPasko) [inline src/main/resources/sql/QuarryHM.sql:14]: write inner join - it is more readable

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Where is my task?

File: src/main/resources/sql/QuarryHM.sql [hunk 2]
```diff
@@ -1,3 +1,9 @@
+--вивести інформацію про клієнтів без повторів, що купили продукти що містять в назві букву "а"
+SELECT * FROM customers cus
```
Comment (SerhiiPasko) [inline src/main/resources/sql/QuarryHM.sql:2]: You have there many times repeats company names

**Review by SerhiiPasko — APPROVED** [review]
Read comment

**Review by Taniann — APPROVED** [review]

**Pull request merged**